sources/packages/web/src/views/aest/institution/InstitutionDetails.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Institution</a
+    >
+  </h5>
+  <full-page-container>
+    <h2 color="primary-color" class="mb-15">
+      {{ initialValue.operatingName }}
+      <designation-status-badge designationStatus="&#10004; DESIGNATED" />
```

Comment:
Can we keep this as text-only?

Reply:
why `designationStatus` is having a hex color code. I am not understanding

Reply:
I tried keeping it only Designated, but the tick symbol is not translated in the component created. You guys can help change it.